apps/fyler/src/aws_cli.erl

```diff
@@ -5,7 +5,7 @@
 -include("../include/log.hrl").
 
 %% API
--export([copy_object/2, copy_object/3, copy_folder/2, copy_folder/3, dir_exists/1]).
+-export([copy_object/2, copy_object/3, copy_folder/2, copy_folder/3, dir_exists/1, instance/1, start_instance/1, stop_instance/1]).
```

Comment:
Line 8 is too long: -export([copy_object/2, copy_object/3, copy_folder/2, copy_folder/3, dir_exists/1, instance/1, start_instance/1, stop_instance/1])..